glass/cosmology.py

```diff
@@ -0,0 +1,25 @@
+"""Module for cosmology.api utilities."""
+
+from typing import Any, Protocol
+
+from numpy.typing import NDArray
+
+import cosmology.api
+
+Array = NDArray[Any]
```

Comment:
Can we move the typing stuff in `TYPE_CHECKING` block or is it required for dynamic resolution?

Reply:
Have moved in what I can

Reply:
Actually, I've had to revert it as tests failed